src/modules/ChannelList/utils.ts

```diff
@@ -133,7 +134,7 @@ const createChannelListQuery = ({
   const params: GroupChannelListQueryParamsInternal = {
     includeEmpty: false,
     limit: 20, // The value of pagination limit could be set up to 100.
-    order: 'latest_last_message', // 'chronological', 'latest_last_message', 'channel_name_alphabetical', and 'metadata_value_alphabetical'
+    order: GroupChannelListOrder.LATEST_LAST_MESSAGE, // 'chronological', 'latest_last_message', 'channel_name_alphabetical', and 'metadata_value_alphabetical'
```

Comment:
We can remove `// 'chronological', 'latest_last_message', 'channel_name_alphabetical', and 'metadata_value_alphabetical'`!